components/ws-manager-mk2/controllers/workspace_controller.go

```diff
@@ -136,7 +136,8 @@ func (r *WorkspaceReconciler) Reconcile(ctx context.Context, req ctrl.Request) (
 	if err = scrubber.Default.Struct(scrubbedStatus); err != nil {
 		log.Error(err, "failed to scrub workspace status")
 	}
-	log.Info("updating workspace status", "status", scrubbedStatus, "podStatus", scrubbedPodStatus)
+
+	log.V(2).Info("updating workspace status", "status", scrubbedStatus, "podStatus", scrubbedPodStatus)
```

Comment:
can we keep this log at info level? This has been one of the most useful logs in debugging any workspace issue as it provides a complete timeline of workspace and pod status

Reply:
+1. This has been super useful.

Reply:
@WVerlaek @Furisto 

Some questions:
- How frequently do we use such log statements for debugging?
- How do we use this in dedicated?
- How do we justify the volume produced?
- How do we inspect the log from substrate?
- WorkspaceStatus contains "everything". What do we really need for debugging?

Reply:
- We use it whenever customers report problems with their workspaces that are lifecycle related (e.g. why did my workspace time out, why is this prebuild still shown as running)
- Not really sure what you are asking. Can you be more specific?
- It is the best debugging tool we have for these kinds of issues
- Stream the log group or export it (/aws/containerinsights/workspace/application)
- Phase and conditions

Reply:
> Stream the log group or export it (/aws/containerinsights/workspace/application)

How are you transforming that to make it readable?

> Phase and conditions

Why we don't just log only that?

Reply:
> How are you transforming that to make it readable?

At the moment we do not apply any transformations

> Why we don't just log only that?
That is already the bulk of what we are logging. Container status would also be nice but we could do without if it is really an issue.

Reply:
> How are you transforming that to make it readable?
There's work planned to make consuming logs in dedicated better (e.g. better querying). For cloud, the verbosity is not really an issue at all due to filtering/querying in gcp

> Why we don't just log only that?
There's more fields that we're interested in, e.g. container statuses, status message, startTime, etc.

Reply:
I think we are not on the same page. I am trying to avoid this:

<img width="748" alt="Screenshot 2023-08-31 at 12 06 20" src="https://github.com/gitpod-io/gitpod/assets/161571/fd9f5a0e-73cc-4425-883c-d45b566cfde5">

In this timeframe, we have peaks of 200 logs per minute of the update, where we have multiple duplications of fields.
In what scenario do we need these logs? What status makes sense to get by default? What's the happy path we can remove?

Reply:
See e.g. the comments in this issue on how we use these logs, they allow us to construct a timeline of the workspace. Without these logs, how would you do that? https://linear.app/gitpod/issue/ENG-252#comment-021b434f

They've meant that we were quickly able to debug workspace-related issues. Without them we're blind on what happened to a workspace. The detailed statuses give valuable context, and the happy path reconciliations also are necessary to build the timeline.